code/modules/mob/living/carbon/life.dm

```diff
@@ -630,9 +630,9 @@ GLOBAL_LIST_INIT(ballmer_windows_me_msg, list("Yo man, what if, we like, uh, put
 	reagents.metabolize(src, can_overdose=FALSE, liverless = TRUE)
 	if(has_trait(TRAIT_STABLEHEART))
 		return
-	adjustToxLoss(8, TRUE,  TRUE)
+	adjustToxLoss(2.5, TRUE,  TRUE) //slightly more than charcoal, anti-toxin or pentetic acid can heal

```

Comment:
doesn't charcoal and antitox stack?

Reply:
Charcoal and antitoxin TOX reduction does stack, but they'll also both remove each other at 1u per tick. You'd need to continually pump them full of both (being careful to avoid OD threshold) in order to keep up.

Tricordrazine gives enough TOX reduction to cover the damage, but doesn't deplete other chemicals. So you could combine that with charcoal and just lose some tricord rather than losing both chems.

There's also earthsblood, which will singlehandedly cover it all.